app/lib/fetchPlaces.ts

```diff
@@ -15,6 +15,8 @@ export type ContributedPlace = {
   googleData?: Place;
   photos?: string[];
   isReportedClosed?: boolean;
+  takeaway: boolean;
+  onPremise: boolean;
```

Comment:
```suggestion
  takeaway?: boolean;
  onPremise?: boolean;
```

Plutôt les mettre optionnels si on veut vraiment typer le JSON à un moment.